Process/Builder/ProcessBuilderInterface.php

```diff
@@ -36,6 +36,8 @@ public function build(ProcessScenarioInterface $scenario);
      *
      * @param string               $name
      * @param string|StepInterface $step Step alias or instance
+     *
+     * @return $this
```

Comment:
For the record: `$this` is perfectly valid and makes sense if you want to indicate a fluent interface. Putting the type there allows you for example immutability.